indico/modules/events/abstracts/controllers/management.py

```diff
@@ -154,6 +154,10 @@ def _process(self):
 
 
 class RHAbstractExportPDF(RHManageAbstractBase):
+    def _checkProtection(self):
```

Comment:
I don't think we need to add this check here. We are extending rom `RHManageAbstractBase`. Doesn't that checks only managers have access? Why wouldn't a manager be able to see reviews?